common/flatpak-transaction.c

```diff
@@ -4758,7 +4758,7 @@ flatpak_transaction_real_run (FlatpakTransaction *self,
         {
           g_autoptr(GBytes) deploy_data = NULL;
           deploy_data = flatpak_dir_get_deploy_data (priv->dir, op->ref,
-                                                     FLATPAK_DEPLOY_VERSION_ANY, NULL, NULL);
+                                                     FLATPAK_DEPLOY_VERSION_CURRENT, NULL, NULL);
```

Comment:
Shouldn’t this be version 4 for eol/eolr info?

Reply:
Doesn't make any functional difference at this point but yeah that's more correct.